src/lib/openapi/spec/application-schema.ts

```diff
@@ -7,31 +7,51 @@ export const applicationSchema = {
     required: ['appName'],
     properties: {
         appName: {
+            description: 'Name of the application',
             type: 'string',
+            example: 'accounting',
         },
         sdkVersion: {
+            description:
+                'Which SDK and version the application reporting uses. Typically represented as `<identifier>:<version>`',
             type: 'string',
+            example: 'unleash-client-java:8.0.0',
         },
         strategies: {
+            description:
+                'Which [strategies](https://docs.getunleash.io/topics/the-anatomy-of-unleash#activation-strategies) the application has loaded. Useful when trying to figure out if your [custom strategy](https://docs.getunleash.io/reference/custom-activation-strategies) has been loaded in the SDK',
             type: 'array',
             items: {
                 type: 'string',
             },
+            example: ['standard', 'gradualRollout', 'mySpecialCustomStrategy'],
         },
         description: {
+            description:
+                'Extra information added about the application reporting the metrics. Only present if added via the Unleash Admin interface',
             type: 'string',
         },
         url: {
+            description:
+                'A link to reference the application reporting the metrics. Could for instance be a GitHub link to the repository of the application',
             type: 'string',
+            example: 'https://github.com/Unleash/unleash-client-proxy-js',
         },
         color: {
+            description: `Css color to be used to color the application's entry in the application list`,
             type: 'string',
+            example: 'red',
         },
         icon: {
+            description: `An URL to an icon file to be used for the applications's entry in the application list`,
             type: 'string',
+            example: 'https://github.com/favicon.ico',
         },
         announced: {
+            description:
+                'Internal to Unleash to allow Unleash to know if the application has already been inserted in a previous batch.',
```

Comment:
What is a batch here? And what does inserting into one mean?

```suggestion
                'Whether the application has been inserted into a previous batch. Used internally by Unleash.',
```

Reply:
Strictly speaking I think we shouldn't expose this, let me clean it up again.